.buildkite/pipeline.yml

```diff
@@ -40,16 +40,16 @@ steps:
           volumes:
             - "/var/run/docker.sock:/var/run/docker.sock"
 
-  - name: "VTAdmin Test"
+  - name: "VTOrc and VTAdmin Test"
     command:
     - apk add g++ make bash curl mysql mysql-client chromium
     - wget https://golang.org/dl/go1.18.3.linux-amd64.tar.gz
     - tar -C /usr/local -xzf go1.18.3.linux-amd64.tar.gz
     - export PATH=$PATH:/usr/local/go/bin
     - rm go1.18.3.linux-amd64.tar.gz
-    - make vtadmin-test
+    - make vtorc-vtadmin-test
     concurrency: 1
-    concurrency_group: 'vtop/vtadmin-test'
+    concurrency_group: 'vtop/vtorc-vtadmin-test'
```

Comment:
Do we plan on adding more components to `vtorc-vtadmin-test`? If so, we could name it to something more generic such as `vitess-components-test`. Same for the Makefile rule, and the test script filename.

Reply:
No, I don't there are any other Vitess components to be added. I think the vtorc-vtadmin name is more descriptive, but I don't have any strong preference between the two

Reply:
makes sense! i don't have strong preference too :)